app/Http/Controllers/Admin/UsersController.php

```diff
@@ -14,6 +17,7 @@ class UsersController extends Controller
      */
     public function index()
     {
-        return view('admin.user.index');
+        $users = User::join('user_infos', 'user_infos.user_id', '=', 'users.id')->get();
```

Comment:
get list phải phân trang chứ em sử dụng paginate của laravel nè em:
https://laravel.com/docs/5.6/pagination
thay vì sử dụng `join` thì em nên sử dụng `with` sẽ clear code hơn đó:
https://laravel.com/docs/5.6/eloquent-relationships#one-to-one